nucypher-core/src/hrac.rs

```diff
@@ -37,6 +37,11 @@ impl HRAC {
         let (hrac, _rest): (GenericArray<u8, U16>, GenericArray<u8, _>) = digest.split();
         Self(hrac.into())
     }
+
+    /// Creates a new HRAC from an array of bytes.
+    pub fn from_bytes(bytes: [u8; 16]) -> Self {
```

Comment:
Sorry for nitpicking, but we already have `ProtocolObject::from_bytes()`, and I'd rather not have them mixed up. What do you think about making it an `impl From<[u8; 16]>` instead?